src/web/edit_load.cc

```diff
@@ -202,11 +202,17 @@ void Web::EditLoad::process()
             resEntry.append_attribute("editable") = false;
         }
         // write resource attributes
-        for (auto&& [key, val] : resItem->getAttributes()) {
-            auto resEntry = resources.append_child("resources");
-            resEntry.append_attribute("resname") = fmt::format(" {}", key).c_str();
-            resEntry.append_attribute("resvalue") = val.c_str();
-            resEntry.append_attribute("editable") = false;
+        for (auto&& attr : ResourceAttributeIterator()) {
+            auto val = resItem->getAttribute(attr);
+            if (!val.empty()) {
+                auto resEntry = resources.append_child("resources");
+                resEntry.append_attribute("resname") = CdsResource::getAttributeDisplay(attr).c_str();
+                auto aVal = resItem->getAttributeValue(attr);
+                if (aVal != val)
+                    aVal = fmt::format("{}<br>{}", aVal, val);
```

Comment:
Should we be putting HTML syntax in our API responses?

Reply:
Surely not, going to move it into ui. Just wanted to share the first draft.

Reply:
Nice, that was my assumption!